doc/modules/linear_model.rst

```diff
@@ -184,6 +184,10 @@ Ridge Complexity
 This method has the same order of complexity as
 :ref:`ordinary_least_squares`.
 
+.. topic:: Examples:
+
+  * :ref:`sphx_glr_auto_examples_linear_model_plot_bayesian_ridge_curvefit.py`
+
```

Comment:
this is in the wrong section. BayesianRidge (which comes later in the document) is different from Ridge